linux-anvil-alma/Dockerfile

```diff
@@ -0,0 +1,93 @@
+ARG DISTRO_ARCH
```

Comment:
Please re-use [linux-anvil-comp7/Dockerfile](https://github.com/conda-forge/docker-images/tree/main/linux-anvil-comp7)

Reply:
Because they are different OSes and versions, which resulted in subtle differences

Also when we run into issues specific to this distro & version, it will be helpful to have them siloed

Most of the things that can be reused already are through refactored scripts that all `Dockerfile`s use

Think it would be better to keep this distinct

Reply:
What are these subtle differences? It would good to have a diff to see what changed for easier review.

Reply:
I'm sorry, but I don't plan to pick up this task

From the conda-forge meeting earlier it sound like others don't care or (in some cases don't want) this change

Would suggest converting this to an issue if we want to discuss further

Reply:
Folks, having it separate makes sense as well as having the diff from the previous one. What can we do here to solve this? Maybe we post a diff of the files here to document that? Do a 2-PR-commit thing overriding the old file, that won't be used and then move it to the dir here, so we document this diff in the git history?

Reply:
Here is the diff

```
0a1
> ARG DISTRO_ARCH
3c4
< FROM ${DISTRO_NAME}:${DISTRO_VER}
---
> FROM --platform=linux/${DISTRO_ARCH} ${DISTRO_NAME}:${DISTRO_VER}
6a8
> ARG DISTRO_ARCH
9c11,12
< ENV DISTRO_NAME=${DISTRO_NAME} \
---
> ENV DISTRO_ARCH=${DISTRO_ARCH} \
>     DISTRO_NAME=${DISTRO_NAME} \
25c28
< RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${DISTRO_VER}
---
> RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
31a35,36
>         glibc-minimal-langpack \
>         glibc-langpack-en \
41,61d45
< # Download and cache new compiler packages.
< # Should speedup installation of them on CIs.
< RUN source /opt/conda/etc/profile.d/conda.sh && \
<     conda activate && \
<     conda create -n test --yes --quiet --download-only \
<         conda-forge::binutils_impl_linux-64 \
<         conda-forge::binutils_linux-64 \
<         conda-forge::gcc_impl_linux-64 \
<         conda-forge::gcc_linux-64 \
<         conda-forge::gfortran_impl_linux-64 \
<         conda-forge::gfortran_linux-64 \
<         conda-forge::gxx_impl_linux-64 \
<         conda-forge::gxx_linux-64 \
<         conda-forge::libgcc-ng \
<         conda-forge::libgfortran-ng \
<         conda-forge::libstdcxx-ng && \
<     conda remove --yes --quiet -n test --all && \
<     conda clean -tiy && \
<     chgrp -R lucky /opt/conda && \
<     chmod -R g=u /opt/conda
< 
65c49
< COPY linux-anvil-cos7-x86_64/entrypoint_source /opt/docker/bin/entrypoint_source
---
> COPY linux-anvil-alma/entrypoint_source /opt/docker/bin/entrypoint_source
```

Reply:
Entrypoint scripts are identical.

Reply:
I agree that different linux distros should be siloed FWIW. Future alma versions should likely use the same image under the assumption that they won't break or change too many things.

Reply:
Let's merge.